app/frontend/src/components/Map.tsx

```diff
@@ -10,21 +10,21 @@ const URL = process.env.REACT_APP_API_BASE_URL;
 mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_KEY!;
 
 const useStyles = makeStyles({
-  root: {
-    position: "relative",
-    height: 200,
-    width: 400,
-  },
   grow: {
-    width: "100%",
     height: "100%",
+    width: "100%",
   },
   map: {
+    height: "100%",
+    left: 0,
     position: "absolute",
     top: 0,
-    left: 0,
     width: "100%",
-    height: "100%",
+  },
+  root: {
+    height: 200,
+    position: "relative",
+    width: 400,
```

Comment:
Moving this here has made 400px width take priority over 100% and `grow` no longer works. Also I'm not sure I like these keys being alphabetical, usually I order them as they appear in the page. What do you think?

Reply:
Ahh crap should have thought of this, I wonder how many things this breaks because the ordering of these matters for CSS. I don't have a problem with ordering by component position. Also I think you mentioned in Slack already, but having `height` and `width` so far apart (imagine there are more properties in between than only `position` in this case) also feels a bit odd to me as they'd be considered related properties to me. Maybe this isn't a good idea and we should just try our best to order it where makes sense...

Reply:
On the other hand, we probably should rely on specificity more than ordering. But yeah the lint rule might be a bit un-subtle?

Reply:
Depends on what you mean by specificity, but I think if we need to use overly specific selectors, it kinda defeats using CSS-in-JS solution a bit, which is supposed to create unique classes so that we shouldn't need highly specific selectors often.

Yeah, unfortunately it's a bit of an all or nothing as it can't be configured to skip checking top level keys in an object. The CSS is basically where most of the warnings come from.

Reply:
Although something like:
```typescript
{
  grow: {
    "div&": { ... }
  }
}
```
probably isn't that bad if we want to keep this? 😅  The styling is still scoped to this component after all...